pkgs/top-level/all-packages.nix

```
@@ -27870,6 +27870,8 @@ with pkgs;
     texinfo = texinfo6_7; # Uses @setcontentsaftertitlepage, removed in 6.8.
   };
 
+  ilspy = callPackage ../applications/misc/ilspy { };
```

```suggestion
  avaloniailspy = callPackage ../applications/misc/avaloniailspy { };
```

This *could* also be uppercase but I consider that bad

And other packagers have gone lowercase for the package name
https://repology.org/project/avaloniailspy/versions